src/lib/Preview.js

```diff
@@ -1802,6 +1852,19 @@ class Preview extends EventEmitter {
         }
     }
 
+    /**
+     * Updates advanced content insights options after props have changed in parent app
+     *
+     * @public
+     * @param {Object} contentInsightsOptions - new content insights options value
+     * @return {void}
+     */
+    updateContentInsightsOptions(contentInsightsOptions) {
+        if (this.options.enableAdvancedContentInsights) {
+            this.previewOptions.contentInsightsOptions = contentInsightsOptions;
```

Comment:
Instead of mutating the object, should we create a new object?
```
this.previewOptions = { ...this.previewOptions, contentInsights: options };
```
Also should we name the options related to content insights as simply `contentInsights` within the `previewOptions`?